tool/ruby_vm/models/typemap.rb

```diff
@@ -16,6 +16,7 @@
   "CDHASH"         => %w[H TS_CDHASH],
   "GENTRY"         => %w[G TS_GENTRY],
   "IC"             => %w[K TS_IC],
```

Comment:
IC should be also renamed.

Reply:
Do you have a suggestion?  `IC` is still used for `opt_getinlinecache` and `opt_setinlinecache` https://github.com/ruby/ruby/blob/e530cfdcdfdc1ffee2771c1e38315859f6a44e5a/insns.def#L1038-L1062

I could rename, but `IC` matches the instruction names.

Reply:
reading related code, we need to refactoring overall. I'll do it so you don't need to change them. thanks.